lib/internal/test_runner/runner.js

```diff
@@ -190,9 +190,9 @@ class FileTest extends Test {
         let directive;
 
         if (skip) {
-          directive = this.reporter.getSkip(node.reason);
+          directive = this.reporter.getSkip(node.reason || true);
         } else if (todo) {
-          directive = this.reporter.getTodo(node.reason);
+          directive = this.reporter.getTodo(node.reason || true);
```

Comment:
so, always?

Reply:
inside this code branch? yes

Reply:
then why the `node.reason ||` at all, if it'll literally always be true?

Reply:
It is a string including a description for the skip, it is not necesaraly a boolean

Reply:
So in `lib/internal/test_runner/tests_stream.js` we are doing [this](https://github.com/nodejs/node/blob/cad0ae702baedb24344be61be9e87c618a080471/lib/internal/test_runner/tests_stream.js#L42-L48). Should we change that to `||` instead of using `||` in this file and then `??` again there?

Reply:
no, since in `tests_stream` we do not want to convert falsy values to `true`, but here it is ok since we are inside an if detecting we are in a skipped test